data/jobs.txt

```diff
@@ -1573,3 +1587,114 @@ mission "Large Rush Delivery [3]"
 		payment
 		payment 42000
 		dialog "You drop off your cargo of <commodity> and collect your payment of <payment>."
+		
+mission "Wealthy Executive Short [0]"
+	name "Transport a wealthy executive  to <planet>"
+	job
+	repeat
+	deadline 0 1
+	description "Bring a wealthy and important executive to <destination> by <date>. He is in a hurry so you will not have time to stop to refuel."
+	to offer
+		random < 10
+		or 
+			has "ship: Flivver"
+			has "ship: Arrow"
+			has "ship: Star Queen"
+	to fail
+		and 
+			not "ship: Flivver"
+			not "ship: Arrow"
+			not "ship: Star Queen"
+	source
+		attributes rich paradise
+		government "Republic" "Free Worlds" "Syndicate" "Quarg" "Neutral"
+	destination
+		distance 2 4
+		attributes rich paradise
+		government "Republic" "Free Worlds" "Syndicate" "Quarg" "Neutral"
+	on complete
+		payment 10000 300
+
+mission "Wealthy Executive Short [1]"
+	name "Transport <bunks> wealthy executives  to <planet>"
```

Comment:
is there generic way to vary words here without making a whole new mission?

it would add a bit of variety to mix between "executives", "aristocrats", "politicians", "VIPs", "celebrities", etc.